example-app/SantokuApp/src/apps/app/contexts/ReactQueryProvider.tsx

```diff
@@ -18,6 +18,16 @@ export const ReactQueryProvider: React.FC<React.PropsWithChildren> = ({children}
       queryCache: defaultQueryCache(queryClient),
       mutationCache: defaultMutationCache(queryClient),
       defaultOptions,
+      // no more errors on the console for tests
+      // https://tanstack.com/query/latest/docs/react/guides/testing#turn-off-network-error-logging
```

Comment:
v5がリリースされてlatestがv5を表示するようになってしまい、このページが存在しなくなったため、
v4固定にしておく
```suggestion
      // https://tanstack.com/query/v4/docs/react/guides/testing#turn-off-network-error-logging
```